### PR TITLE
Follow-up to Ceph end-to-end example (PR#1772)

### PR DESCRIPTION
**Pull request by ahardin-rh**
Follow-up to https://github.com/openshift/openshift-docs/pull/1772

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@adellape @tnguyen-rh @bfallonf @tpoitras PTAL

File: install_config/storage_examples/ceph_example.adoc [hunk 1]
```diff
@@ -12,34 +12,41 @@ toc::[]
...
+link:../persistent_storage/persistent_storage_ceph_rbd.html[Persistent Storage
+Using Ceph Rados Block Device] provides an explanation of persistent volumes
+(PVs), persistent volume claims (PVCs), and using Ceph RBD as persistent
+storage.
 
 [NOTE]
 ====
-All `oc ...` commands are executed on the OpenShift master node.
+All `oc ...` commands are executed on the {product-title} master node.
 ====
```
Comment (bfallonf) [inline install_config/storage_examples/ceph_example.adoc]: Same nit from the other PR. Should master node be "master host" or "all-in-one host"?

File: install_config/storage_examples/ceph_example.adoc [hunk 2]
```diff
@@ -12,34 +12,41 @@ toc::[]
...
+[[using-ceph-rbd-installing-the-ceph-common-package]]
+== Installing the ceph-common Package
+The *ceph-common* library must be installed on *all schedulable* {product-title} nodes:
 
 [NOTE]
 ====
-Often the OpenShift master node is not used to run pod workloads, and, thus, is not included as a schedulable node.
+The {product-title} master node is not often used to run pod workloads and,
+thus, is not included as a schedulable node.
 ====
```
Comment (bfallonf) [inline install_config/storage_examples/ceph_example.adoc]: ^ Here too

File: install_config/storage_examples/ceph_example.adoc [hunk 3]
```diff
@@ -78,9 +83,10 @@ ceph-secret   Opaque    1         23d
 ----
 ====
 
+[[using-ceph-rbd-creating-the-persistent-volume]]
 == Creating the Persistent Volume
 The persistent volume (PV) file is defined next before creating the PV object in
```
Comment (bfallonf) [inline install_config/storage_examples/ceph_example.adoc]: @ahardin-rh I like the edit you made to this sentence in the other PR :smiley:

**Comment by bfallonf** [comment]
@ahardin-rh A few comments, but otherwise LGTM.

**Comment by ahardin-rh** [comment]
@bfallonf Thanks! Fixed! Mergin'!  :rocket:

**Comment by ahardin-rh** [comment]
[rev_history]
|link:../install_config/storage_examples/ceph_example.html[Storage Examples -> Complete Example Using Ceph RBD]
|New topic providing end-to-end example of using an existing Ceph cluster as an {product-title} persistent store.

**Pull request merged**